src/change_stream.ts

```diff
@@ -210,7 +210,7 @@ export interface ChangeStreamDocumentCollectionUUID {
    * **NOTE:** collectionUUID will be converted to a NodeJS Buffer if the promoteBuffers
    *    flag is enabled.
    *
-   * @since 6.1.0
+   * @sinceServerVersion 6.1.0
```

Comment:
Neal pointed out that `@since` is potentially confusing, because we could refer to the driver or the server version.  I created a new tag to make it more clear